### PR TITLE
version bumps for reflex-platform v0.9.2.0

### PR DESCRIPTION
**Pull request by rubenmoor**
to put it bluntly, I don't know what I am doing. I upgraded to obelisk 1.0.0.0 and with the small changes below, everythings works fine. I don't know how or why `HasWebView` disappeared from `reflex-dom`.

### CODE REVIEW

File: src/Servant/Common/BaseUrl.hs [hunk 1]
```diff
@@ -32,7 +32,7 @@ import           Reflex.Dom.Core
 import           Text.Read
 
 
-type SupportsServantReflex t m = (Reflex t, TriggerEvent t m, PerformEvent t m, HasWebView (Performable m), MonadJSM (Performable m))
+type SupportsServantReflex t m = (Reflex t, TriggerEvent t m, PerformEvent t m, MonadJSM (Performable m))
```
Comment (imalsogreg) [inline src/Servant/Common/BaseUrl.hs:35]: Looking at `reflex-dom-core`'s changelog, it looks like `HasWebView` was deleted just a couple weeks ago in the bump to 0.7.0. I think that means dropping them from `servant-reflex` forces us onto exactly `reflex-dom-core-0.7.0`.

Comment (imalsogreg) [reply src/Servant/Common/BaseUrl.hs:35]: If you're curious about that change, here's the MR: https://github.com/reflex-frp/reflex-dom/pull/429/files

**Comment by imalsogreg** [comment]
> I don't know what I am doing

It looks like you bumped versions, and followed compiler errors to drop newly-deleted identifiers. This is about as much "knowing what you're doing" as I usually have :)

Thanks for the PR. Do you mind raising the lower bounds on `reflex-dom-core` to `>= 0.7.0`, and then I'll merge?

File: src/Servant/Common/BaseUrl.hs (hunk 1 above)
Comment (rubenmoor) [reply src/Servant/Common/BaseUrl.hs:35]: got it!

**Comment by rubenmoor** [comment]
> > I don't know what I am doing
> 
> It looks like you bumped versions, and followed compiler errors to drop newly-deleted identifiers. This is about as much "knowing what you're doing" as I usually have :)
> 
> Thanks for the PR. Do you mind raising the lower bounds on `reflex-dom-core` to `>= 0.7.0`, and then I'll merge?

will do! and thanks for servant-reflex

**Comment by rubenmoor** [comment]
> > I don't know what I am doing
> 
> It looks like you bumped versions, and followed compiler errors to drop newly-deleted identifiers. This is about as much "knowing what you're doing" as I usually have :)
> 
> Thanks for the PR. Do you mind raising the lower bounds on `reflex-dom-core` to `>= 0.7.0`, and then I'll merge?

done

**Comment by imalsogreg** [comment]
Thanks a lot!

**Pull request merged**